templates/sudoers.j2

```diff
@@ -0,0 +1,4 @@
+# Managed by Ansible
```

Comment:
There's a variable for that `{{ ansible_managed }}`

Reply:
I didn't know it, thanks!